network-store-server/src/main/resources/db/changelog/changesets/changelog_20241121T110000Z.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <changeSet id="2986331049275-5" author="lesoteti">
+        <createTable tableName="newtemporarylimits">
+            <column name="networkuuid" type="VARCHAR(255)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="newtemporarylimits_pkey"/>
+            </column>
+            <column name="variantnum" type="INTEGER">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="newtemporarylimits_pkey"/>
+            </column>
+            <column name="equipmentid" type="VARCHAR(255)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="newtemporarylimits_pkey"/>
+            </column>
+            <column name="equipmenttype" type="VARCHAR(255)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="newtemporarylimits_pkey"/>
```

Comment:
Not sure equipmenttype should be in the primaryKey. In iidm, we can't have two equipments with the same id. equipmentid should be enough

Reply:
yes but maybe it can optimize the request if we want the limits only for line or for twt what do you think ?

Reply:
You should build the indexes depending on the requests you make. Look at the columns you use in you WHERE close and build the corresponding indexes

Reply:
ok